common/lib/xmodule/xmodule/tests/test_library_tools.py

```diff
@@ -0,0 +1,28 @@
+"""
+Tests for library tools service.
+"""
+from mock import patch
+from xmodule.library_tools import LibraryToolsService
+from xmodule.modulestore.tests.factories import LibraryFactory
+from xmodule.modulestore.tests.utils import MixedSplitTestCase
+
+
+class LibraryToolsServiceTest(MixedSplitTestCase):
```

Comment:
Do you also like to add a test for success scenario for `list_available_libraries`?